packages/core/content-manager/admin/src/preview/index.ts

```diff
@@ -8,10 +9,17 @@ const previewAdmin = {
   bootstrap(app) {
     // TODO: Add license registry check when it's available
     if (!window.strapi.future.isEnabled(FEATURE_ID)) {
-      return {};
+      return;
+    }
+
+    const contentManagerPluginApis = app.getPlugin('content-manager').apis;
+
+    if (
+      'addEditViewSidePanel' in contentManagerPluginApis &&
+      typeof contentManagerPluginApis.addEditViewSidePanel === 'function'
+    ) {
```

Comment:
It is the API indeed. It should always exist. The annoying thing is that TS doesn't know it, since the type of .apis is `Record<string, unknown>`. So I copied what was done in releases to convince TS it's a function, but I could also use `!` if you prefer

Reply:
oh I see, its fine by me! at least if we keep that Record type it looks more correct to do it like that